### DISCUSSION TITLE
How to stop the prompt pop up saying to input the date and time when I am using the QR code with timecode on it

### DISCUSSION THREAD

**Discussion by Tori456** [gopro/labs#769] [Q&A]
Good Morning :) I just wondered if there is a way to override the pop up asking me to set up the date and time so the go pro just goes off the QR codes time code? It would be super helpful as im using 18 go pros and its time consuming to manually set them all up. 

Thank you again!
Tori :)

**Comment by dnewman-gpsw** [comment]
I believe !MBYPS=1 will mute all of the UI nagging popups.

**Reply by Tori456** [reply]
Thank you for your responce :) I just tried this and unfortunately the pop up happened again after :(

**Reply by dnewman-gpsw** [reply]
Which camera model and firmware?

My answer are almost always assume the latest if you don't specify.

**Reply by Tori456** [reply]
Hey, it's for hero 11 and with the latest firmware

**Reply by dnewman-gpsw** [reply]
HERO11 should support this.  It is seems that once the flag is set (for the popup) the !MBYPS=1 may not suppress it.  If you set the time, the flag is reset, the message should not appear again.
